styx-client/src/main/java/com/spotify/styx/client/StyxWorkflowClient.java

```diff
@@ -74,6 +75,18 @@
    */
   CompletionStage<WorkflowState> workflowState(final String componentId, final String workflowId);
 
+  /**
+   * Get execution data of an instance of a {@link Workflow}
+   *
+   * @param componentId component id
+   * @param workflowId  workflow id
+   * @param parameter   parameter
+   * @return the {@link WorkflowInstanceExecutionData}
+   */
+  CompletionStage<WorkflowInstanceExecutionData> workflowInstance(final String componentId,
```

Comment:
Should this be called `workflowInstanceExecutions` instead?

Reply:
Yeah, I wasn't too sure about that. I end up with this name because in the API (https://spotify.github.io/styx/api.html#workflows-workflow-instance-get) says "Get Workflow Instance" in the description of the endpoint... :$

Reply:
Renamed @danielnorberg !